models/article.model.js

```diff
@@ -10,3 +10,31 @@ exports.selectArticleById = (id) => {
       return rows[0];
     });
 };
+
```

Comment:
This is a really creative solution, however, please be aware that you could have done this with just one db query! If you wish to refactor, take a look at ALIAS within SQL and how this could be implemented. 👍